tools/packaging/test262.py

```diff
@@ -218,7 +218,7 @@ def HasUnexpectedOutcome(self):
     if self.case.IsAsyncTest():
        return self.AsyncHasFailed() or self.HasFailed()
     elif self.case.IsNegative():
-       return not (self.HasFailed() and self.case.NegativeMatch(self.GetErrorOutput()))
+       return not (not self.HasFailed() and self.case.NegativeMatch(self.GetErrorOutput()))
```

Comment:
We could, instead, do:
return self.HasFailed() or not self.case.NegativeMatch(self.GetErrorOutput())